### PR TITLE
Add index unittests for query node

### LINKED ISSUE #7564: Add unittests for query node

### PR DESCRIPTION
**Pull request by bigsheeper**
Signed-off-by: bigsheeper <yihao.dai@zilliz.com>

Resolves: #7564

### CODE REVIEW

File: internal/querynode/index_loader.go [hunk 1]
```diff
@@ -153,85 +101,6 @@ func (loader *indexLoader) printIndexParams(index []*commonpb.KeyValuePair) {
...
-		return 0, 0, err
-	}
-	fieldID, err := strconv.ParseInt(ids[1], 10, 64)
-	if err != nil {
-		return 0, 0, err
-	}
-	return collectionID, fieldID, nil
-}
-
-func (loader *indexLoader) updateSegmentIndexStats(segment *Segment) error {
```
Comment (czs007) [inline internal/querynode/index_loader.go:192]: better not to delete this function, is it not easy to add a single test?

Comment (bigsheeper) [reply internal/querynode/index_loader.go:192]: This function has been deprecated, query node dosen't need to send index stats to msgstream anymore.

Comment (bigsheeper) [reply internal/querynode/index_loader.go:192]: I would comment these codes

**Pull request merged**